docs/rules/explicit-length-check.md

```diff
@@ -0,0 +1,21 @@
+# Enforce explicitly checking the length of an array
+
+Enforce explicitly checking the length of an array in an `if` condition, rather than checking the truthiness of the length.
+
+## Fail
+
+```js
+if (array.length) {}
+
+if (!array.length) {}
+```
+
+
+## Pass
+
+```js
+if (array.length > 0) {}
+if (array.length !== 0) {}
+
```

Comment:
I was trying to map the previous incorrect code to the right correct code, by delimiting them by an empty line. Do you prefer the line to be removed?